code/modules/mob/living/simple_mob/subtypes/humanoid/mercs/mercs.dm

```diff
@@ -258,9 +258,80 @@
 
 	corpse = /obj/effect/landmark/mobcorpse/syndicatecommando
 
+	loot_list = list(/obj/item/gun/projectile/automatic/c20r = 100)
+	base_attack_cooldown = 5 // Two attacks a second or so.
+	reload_max = 20
+
 /mob/living/simple_mob/humanoid/merc/ranged/space/Process_Spacemove(var/check_drift = 0)
 	return
 
+// suppressors are just assholes and are intended to be a piss poor experience for everyone on both sides
+
+/datum/ai_holder/simple_mob/merc/ranged/suppressor
+	respect_alpha = FALSE // he really just shoots you
+	vision_range = 10 // plutonia experience
+
+/mob/living/simple_mob/humanoid/merc/ranged/space/suppressor // adminspawn only, and also Probably Going To Kill The Unprepared
+	name = "syndicate suppressor"
+	desc = "Geeze, weren't shotgun ops bad enough? At least when you fade these jerks you get a flashbang to the face."
+	icon_state = "syndi-ranged-space-sup"
+	icon_living = "syndi-ranged-space-sup"
+	armor = list(melee = 80, bullet = 65, laser = 50, energy = 15, bomb = 80, bio = 100, rad = 100) // this is the merc rig's stats
+	ai_holder_type = /datum/ai_holder/simple_mob/merc/ranged/suppressor
+	say_list_type = /datum/say_list/merc/elite
+	projectiletype = /obj/item/projectile/bullet/pistol/medium/ap/suppressor // it's high velocity
+	projectilesound = 'sound/weapons/doompistol-perkristian.ogg' // converted from a wav taken from http://www.perkristian.net/game_doom-sfx.shtml
+	base_attack_cooldown = 3 // three? attacks a second
+	reload_max = 30 // extended mags
+	special_attack_charges = 5
+	loot_list = list()
+	corpse = null
+	var/deathnade_path = /obj/item/grenade/flashbang
+
+/mob/living/simple_mob/humanoid/merc/ranged/space/suppressor/death()
+	// you thought killing him would be the least of your worries?
+	// think again
+	var/obj/item/grenade/banger = new deathnade_path(get_turf(src))
+	banger.throw_at(ai_holder.target, 9, 9, null)
+	if(istype(banger, /obj/item/grenade))
+		banger.det_time = 2
+		banger.activate(null)
+	..()
+
+/mob/living/simple_mob/humanoid/merc/ranged/space/suppressor/elite // really reconsider why you're spawning this dude
+	name = "elite mercenary suppressor"
+	desc = "Geeze, weren't normal suppressors bad enough? At least if you fade this jerk, you'll still have an awful time anyway."
+	icon_state = "syndi-ranged-space-sup-elite"
+	icon_living = "syndi-ranged-space-sup-elite"
+	armor = list(melee = 80, bullet = 70, laser = 55, energy = 15, bomb = 80, bio = 100, rad = 100) // see code for military hardsuit
+	projectiletype = /obj/item/projectile/bullet/pistol/medium/ap/suppressor/hitscan
```

Comment:
As said previously drop the hitscan. No one wants it. Use the high speed suppressor round you already created.

Reply:
gave elites an 8x speed bullet